src/components/pages/SearchResultsPage.js

```diff
@@ -53,19 +83,65 @@ const SearchResultsPage = () => {
             <h1 className="text-white font-bold text-5xl">
               {location.state.value}
             </h1>
-            <h2 className="text-white font-semibold text-3xl mt-2">$138.08</h2>
+            <h2 className="text-white font-semibold text-3xl mt-2">
+              ${stockData[0].recentPrice}
+            </h2>
             <p className="text-bright-green font-medium text-md mt-2">
-              +$4.61 (+3.22%)
-              <p className="inline text-light-gray font-light"> Today</p>
+              {stockData[0].recentPrice - stockData[0].open >= 0 ? "+" : "-"} $
```

Comment:
Jesse will return this  , its fine for now but just noting that time will just use whats returned from thendpoint not recent price - open.